datasets/users.js

```diff
@@ -0,0 +1,55 @@
+let users = [
+  {
+    id: 1,
+    name: "Tommy",
+  },
+  {
+    id: 2,
+    name: "Ben",
+  },
+];
+
+function getUsers() {
+  return users;
+}
+
+function getUserByID(id) {
+  return users.find((user) => user.id == id);
+}
+
+function addUser(name) {
+  //Generate ids until a unique id is found
+  let id = Math.random().toString(10).slice(-5);
```

Comment:
Did we want to generate a random id for the users like this as well?
We can discuss this in the data modeling meeitng. :)